docs/docs/getting_started.md.in

```diff
@@ -226,7 +241,7 @@ ILLIXR currently supports both OpenGL and Vulkan backends (indicated as `gl` and
 
 ## Rationale
 
--  The current system can use profile files to control everything from the build to running ILLIXR, inkeeping with the [DRY principle][1].
```

Comment:
To comment on line 240 - mention that OpenGL support is deprecating.

Reply:
added a warning admonition